lib/xcode/install.rb

```diff
@@ -49,8 +49,8 @@ def current_symlink
     end
 
     def download(version, progress, url = nil)
-      return unless exist?(version) || url
-      xcode = seedlist.find { |x| x.name == version }
+      return unless url || exist?(version)
+      xcode = seedlist.find { |x| x.name == version } unless url
```

Comment:
This shouldn't be needed as the previous line already returns if `url` is non-nil or am I missing something?